src/main/java/com/datastax/loader/CqlSchemaFuzzyMatcher.java

```diff
@@ -0,0 +1,63 @@
+package com.datastax.loader;
+
+import me.xdrop.fuzzywuzzy.FuzzySearch;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * Created by sebastianestevez on 12/6/16.
+ */
+
+public class CqlSchemaFuzzyMatcher {
+    public CqlSchemaFuzzyMatcher() {
+    }
+
+    public String match(String keyspace, String table, BufferedReader reader, String delimiter, CqlDelimParser cdp) {
+        String schemaString = keyspace.replaceAll("\"","")+ "." + table.replaceAll("\"","") + "(";
+        List<String> columnNames = cdp.getColumnNames();
+        try {
+            String header = reader.readLine();
+            //this didn' work
+            //System.out.println(header);
+            //List<Object> headerList = cdp.parse(header);
+            if (delimiter == null){
+                delimiter = ",";
+            }
+            List<String> headerList = Arrays.asList(header.split(delimiter));
+            String[] resultColumns = new String[headerList.size()];
+            if (headerList.size()> columnNames.size()){
+                System.err.println("There are too many columns in the header.");
+                System.exit(1);
+            }
+            int i =0;
+            for (Iterator<String> headerIterator = headerList.iterator(); headerIterator.hasNext();){
+                int currentRatio = 0;
+                String currentColumn = "";
+                String headerField = (String)headerIterator.next();
+                for (Iterator<String> columnIterator = columnNames.iterator(); columnIterator.hasNext(); ) {
+                    String column= columnIterator.next().replaceAll("\"","");
+                    int newRatio = FuzzySearch.ratio(headerField,column);
+                    if (newRatio > currentRatio) {
+                        currentColumn = column;
+                        currentRatio = newRatio;
+                    }
+                }
+                if (currentRatio < 100){
+                    System.err.println("The match between header "+ headerField + " and column " +  currentColumn + " was not exact.");
+                    System.err.println("Proceeding with fuzzy match of score: " + currentRatio);
+                }
+                resultColumns[i]=currentColumn;
+                i++;
+                columnNames.remove(columnNames.contains(currentColumn));
```

Comment:
__columnNames.contains(...)__ returns a __boolean__;  passing any boolean to __columnNames.remove(...)__ will have no effect

Perhaps this was meant to remove __currentColumn__ from __columnNames__ if it exists in the list. In this case it is fine to just call __columnNames.remove(currentColumn)__